pkg/lib/cockpit-components-plot.jsx

```diff
@@ -195,14 +195,14 @@ function value_ticks(data, config) {
         const unit_str = config.formatter(unit, config.base_unit, true)[1];
 
         return {
-            ticks: ticks,
+            ticks,
             formatter: (val) => config.formatter(val, unit_str, true)[0],
             unit: unit_str,
             max: ticks[ticks.length - 1]
         };
     } else {
         return {
-            ticks: ticks,
+            ticks,
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-210817-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)